i18n/en.pot

```diff
@@ -209,15 +211,6 @@ msgstr ""
 msgid "Antigens"
 msgstr ""
 
-msgid "Population distribution (%)"
-msgstr ""
-
-msgid "Campaign Population Distribution"
-msgstr ""
-
-msgid "Missing"
-msgstr ""
-
```

Comment:
Those literals should not be removed. I think you need to do a "yarn link" our custom d2-i18n-extract. See:

https://github.com/EyeSeeTea/vaccination-app/issues/81#issuecomment-484438882